Havit.Blazor.Components.Web.Bootstrap/wwwroot/defaults.css

```diff
@@ -79,6 +80,9 @@
 	--hx-context-menu-button-hover-background: 						var(--bs-secondary-bg);
 	--hx-context-menu-item-icon-margin: 							0 .5rem 0 0;
 
+	/* HxDropdown */
+	--hx-dropdown-menu-item-icon: 									0 .5rem 0 0;
```

Comment:
This should be named `--hx-dropdown-menu-item-icon-margin` (see the `HxContextMenuItem` variable few lines above).